test/mailers/patient_mailer_test.rb

```diff
@@ -257,6 +257,27 @@ def twilio_error
     end
   end
 
+  %i[assessment_sms_weblink].each do |mthd|
+    test "#{mthd} success histories for purged HOH" do
```

Comment:
I'm assuming you're planning on adding more tests for the unsuccessful histories too correct?